.cargo/config

```diff
@@ -1,6 +1,6 @@
 [alias]
-wasm = "build --release --target wasm32-unknown-unknown"
-wasm-debug = "build --target wasm32-unknown-unknown"
+wasm = "build --release --lib --target wasm32-unknown-unknown"
+wasm-debug = "build --lib --target wasm32-unknown-unknown"
```

Comment:
I needed to do this because `ci/bootstrap-dev` has a main file that I cannot easily exclude for the wasm runtime like we do the the `lib.rs` files.